sklearn/_loss/tests/test_loss.py

```diff
@@ -981,6 +982,44 @@ def test_binomial_and_multinomial_loss(global_random_seed):
     )
 
 
+def test_binomial_vs_alternative_formulation():
```

Comment:
Given the size of this PR, can this test be moved to it's own PR?

Reply:
Yes, I'll do that. Though, without the context of this PR, it would have been an odd PR.

Reply:
Done in #26298.